src/Client/Extensions/HttpClientUserInfoExtensions.cs

```diff
@@ -26,8 +26,14 @@ public static async Task<UserInfoResponse> GetUserInfoAsync(this HttpMessageInvo
         if (request.Token!.IsMissing()) throw new ArgumentNullException(nameof(request.Token));
 
         var clone = request.Clone();
-
+        
+        clone.ClientId = string.Empty;
+        clone.ClientSecret = string.Empty;
+        clone.ClientAssertion = new ClientAssertion();
+        clone.Parameters = new Parameters();
+        
         clone.Method = HttpMethod.Get;
```

Comment:
It seems like the real issue is that the method is always a GET even if there is content in the form of client auth or other parameters. I think it is probably confusing to discard those extra parameters, and so if we want that to not be allowed at all, then it should be an explicit error, and the fundamental problem is the fact that it is always a GET. (Spec does allow POST).